cclib/method/nuclear.py

```diff
@@ -100,6 +100,91 @@ def stoichiometry(self):
             sign = "+" if self.data.charge > 0 else "-"
             formula += "(%s%i)" % (sign, magnitude)
         return formula
+    
+    def get_distances(self, atomcoords_index = -1):
+        '''Returns a matrix with distances between i_th and j_th atoms at index [i][j], where i < j
+        *Other indices will give wrong values*'''
+        coords = self.data.atomcoords[atomcoords_index]
+        number_of_atoms = self.data.atomcoords.size[1]
+        distance_matrix = np.empty((number_of_atoms, number_of_atoms))
+        i = 0
+        j = 1
+        while(i < number_of_atoms):
```

Comment:
In Python it is more more conventional to use `for` loops for these types of things.

Reply:
Yes. I guess that makes sense. I have made changes in the new commit.